app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/about/AboutScreen.kt

```diff
@@ -507,5 +507,5 @@ private fun ApiDetailsSection() {
 @Preview(uiMode = Configuration.UI_MODE_NIGHT_YES, showBackground = true)
 @Composable
 fun PreviewAboutScreen() {
-    SampleAppTheme { AboutScreen() }
+    SampleAppTheme { AboutScreen {} }
```

Comment:
not sure this needs curly brackets

Reply:
We don't need to support the `onBackPressed` in a preview, so I've left it as an empty lambda